DittoExporter/src/main/java/live/ditto/exporter/ExportDialog.kt

```diff
@@ -0,0 +1,114 @@
+package live.ditto.exporter
+
+import androidx.activity.compose.rememberLauncherForActivityResult
+import androidx.activity.result.contract.ActivityResultContracts.CreateDocument
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Button
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import androidx.core.net.toFile
+import kotlinx.coroutines.launch
+import java.io.File
+import java.io.FileInputStream
+
+/**
+ * A customized AlertDialog to export the given [file]
```

Comment:
Change to `[File]` so it references the `java.io.File`

Reply:
It references the property `file` not the data type

Reply:
oh the issue is the property name was change from `file` to `fileProvider` and for whatever reason AS did not change it.